src/tests/ecalc_cli/snapshots/test_app/test_json_advanced_model/test_json_advanced_model_v3.json

```diff
@@ -42892,28 +43004,28 @@
         "unit": "GWh",
         "values": [
           0.0,
-          58.38442,
-          140.5835,
-          280.7829,
-          420.9824,
-          561.1818,
-          706.233,
-          855.3434,
-          1009.684,
-          1279.885,
-          1550.826,
-          1705.166,
-          1854.277,
-          1989.377,
-          2124.848,
-          2259.949,
-          2395.05,
-          2530.15,
-          2665.621,
-          2800.722,
-          2935.822,
-          3070.923,
-          3206.394
+          77.21123,
```

Comment:
should the changed result here be 77.21123 * 0.75 = result before?

Reply:
Not as far as I know, haven´t looked into what the expected change should be. The reason for change in results is that compressor charts have changed from predefined charts to generic.